src/js/components/contact_us/contactUs.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import logo from '../../../img/logo-colored.png';
+import {Link, browserHistory} from 'react-router';
+
+class ContactUs extends React.Component {
+  state = {
+    email: "",
+    message: ""
+  }
+
+  render() {
+    let key = 0;
+    return (
+      <div className="container eligibility-check">
+        <div className="header">
+          <img src={logo} />
+          <Link to="/">
+            <i className="fa fa-window-close" aria-hidden="true" />
+          </Link>
+        </div>
+
+        <div className="page-title">
+          <h2>We'll love to hear from you</h2>
+          <h5>Please send us a message :)</h5>
+        </div>
+
+        <content className="wrapper">
+          <form>
+          {/*<div className="progress">
+            <div className="progress-bar bg-success" role="progressbar" aria-valuenow="70"
+                 aria-valuemin="0" aria-valuemax="100">
+              <span className="sr-only">70% Complete</span>
+            </div>
+          </div>*/}
```

Comment:
If you don't need this code, then it should be removed instead of commenting it out.